examples/Bela/bela_test_cases.scd

```diff
@@ -0,0 +1,312 @@
+Server.default = Server.remote("belaServer", NetAddr("192.168.7.2", 57110));
```

Comment:
is there a reason that alll the examples use `SynthDef().send(s)` instead of the more usual `SynthDef().add` ?

Reply:
not sure. @sensestage authored these 5 years ago ... maybe a stylistic choice?

@telephon would you recommend changing to `.add` ?

Reply:
If the default server is bela anyway, then yes!

Reply:
79a13d444